### PR TITLE
Styling/admin page

### PR DESCRIPTION
**Pull request by 545zhou**

### CODE REVIEW

File: common/templates/common/header.html [hunk 1]
```diff
@@ -50,7 +48,7 @@
                         <li><a href="/prereg">Preregistration Challenge</a></li>
                     </ul>
                 </li>
-                 <li class="dropdown 
+                 <li class="dropdown
                                      ">
```
Comment (atelic) [inline common/templates/common/header.html]: This closing quote should be on the same line as the opening one.

File: requirements.txt [hunk 2]
```diff
@@ -14,6 +14,7 @@ django-toolbelt==0.0.1
 django-treebeard==4.0.1
 djangorestframework==3.4.6
 docutils==0.12
+elasticsearch==1.2.0
```
Comment (atelic) [inline requirements.txt:17]: Do any changes to the `README.md` need to be made to explain getting elastic working?

**Review by atelic — CHANGES_REQUESTED** [review]

File: requirements.txt (hunk 2 above)
Comment (545zhou) [reply requirements.txt:17]: Yes. I will do that soon.

Comment (NyanHelsing) [reply requirements.txt:17]: the README is pretty out of date in general. I don't think we'll use heroku as a permanent solution, so don't worry about heroku-specific things much until we know the deploy better.

**Pull request merged**